### PR TITLE
LEDGER pov

### PR DESCRIPTION
**Pull request by williamdemeo**
# Description

Addresses third item of issue #467.

# Checklist

- [x] Commit sequence broadly makes sense and commits have useful messages
- [x] Any semantic changes to the specifications are documented in `CHANGELOG.md`
- [x] Code is formatted according to [CONTRIBUTING.md](https://github.com/input-output-hk/formal-ledger-specifications/blob/master/CONTRIBUTING.md)
- [x] Self-reviewed the diff

### CODE REVIEW

**Review by WhatisRT — APPROVED** [review]
LGTM

**Review by WhatisRT — CHANGES_REQUESTED** [review]
Sorry, I approved it too soon

File: src/Ledger/Ledger/Properties.agda [hunk 1]
```diff
@@ -99,39 +99,69 @@ Computational-LEDGERS = it
...
+  ( indexedSumᵛ'-∪  :  {A : Type} ⦃ _ : DecEq A ⦄ (m m' : A ⇀ Coin)
+                       → disjoint (dom m) (dom m') → getCoin (m ∪ˡ m') ≡ getCoin m + getCoin m' )
+  ( sumConstZero    :  {A : Type} ⦃ _ : DecEq A ⦄ {X : ℙ A} → getCoin (constMap X 0) ≡ 0 )
+  ( res-decomp      :  {A : Type} ⦃ _ : DecEq A ⦄ (m m' : A ⇀ Coin)
+                       → (m ∪ˡ m')ˢ ≡ᵉ (m ∪ˡ (m' ∣ dom (m ˢ) ᶜ))ˢ )
+  ( getCoin-cong    :  {A : Type} ⦃ _ : DecEq A ⦄ (s : A ⇀ Coin) (s' : ℙ (A × Coin)) → s ˢ ≡ᵉ s'
+                       → indexedSum' proj₂ (s ˢ) ≡ indexedSum' proj₂ s' )
+  ( ≡ᵉ-getCoinˢ     :  {A A' : Type} ⦃ _ : DecEq A ⦄ ⦃ _ : DecEq A' ⦄ (s : ℙ (A × Coin)) {f : A → A'}
+                       → InjectiveOn (dom s) f → getCoin (mapˢ (map₁ f) s) ≡ getCoin s )
+  ( constNetworkId  :  (wdls : RwdAddr ⇀ Coin) → ∀[ a ∈ dom (wdls ˢ) ] RwdAddr.net a ≡ NetworkId )
```
Comment (WhatisRT) [inline src/Ledger/Ledger/Properties.agda:121]: This follows from the following line in `UTXO-inductive`:
```
∙ ∀[ a ∈ dom txwdrls ]          a .RwdAddr.net  ≡ networkId
```

File: src/Ledger/Certs/Properties.agda [hunk 2]
```diff
@@ -207,20 +208,22 @@ module _  { indexedSumᵛ'-∪ :  {A : Type} ⦃ _ : DecEq A ⦄ (m m' : A ⇀ C
...
+  module CERTSpov
+    -- TODO: prove some or all of the following assumptions, used in roof of `CERTBASE-pov`.
+    ( sumConstZero    :  {A : Type} ⦃ _ : DecEq A ⦄ {X : ℙ A} → getCoin (constMap X 0) ≡ 0 )
+    ( res-decomp      :  {A : Type} ⦃ _ : DecEq A ⦄ (m m' : A ⇀ Coin)
+                         → (m ∪ˡ m')ˢ ≡ᵉ (m ∪ˡ (m' ∣ dom (m ˢ) ᶜ))ˢ )
+    ( getCoin-cong    :  {A : Type} ⦃ _ : DecEq A ⦄ (s : A ⇀ Coin) (s' : ℙ (A × Coin)) → s ˢ ≡ᵉ s'
+                         → indexedSum' proj₂ (s ˢ) ≡ indexedSum' proj₂ s' )
+    ( ≡ᵉ-getCoinˢ     :  {A A' : Type} ⦃ _ : DecEq A ⦄ ⦃ _ : DecEq A' ⦄ (s : ℙ (A × Coin)) {f : A → A'}
+                         → InjectiveOn (dom s) f → getCoin (mapˢ (map₁ f) s) ≡ getCoin s )
+    ( constNetworkId  :  (wdls : RwdAddr ⇀ Coin) → ∀[ a ∈ dom (wdls ˢ) ] RwdAddr.net a ≡ NetworkId )
```
Comment (WhatisRT) [inline src/Ledger/Certs/Properties.agda:226]: You need to specialize this to `CertEnv.wdrls Γ`

File: src/Ledger/Ledger/Properties.agda (hunk 1 above)
Comment (williamdemeo) [reply src/Ledger/Ledger/Properties.agda:121]: The problem here is that we have multiple `NetworkId`s preventing a unification in one of the proofs.  I'll remove one of the `NetworkId`s from `globalConstants`.

**Review by WhatisRT — APPROVED** [review]
LGTM

**Pull request merged**